ambari-logsearch-web/src/assets/i18n/en.json

```diff
@@ -377,6 +377,22 @@
     "page": "Page",
     "query": "Query",
     "users": "Users"
+  },
+
+  "userSettings": {
```

Comment:
Declaring these props in nested form (i.e., `"userSettings": {"modalTitle": {"displayShortHostName": {"description": "..."}}} ` instead of `"userSettings.modalTitle.displayShortHostName."description": "..."` makes application harder to debug

Reply:
In the same time it makes the file more readable. However I felt that it can be confused to have two different approach. I change this and make the whole language file format as it was used before.

Reply:
In the same time the file become more readable and the file size is smaller with nested props.

Reply:
If so, it makes sense to have the whole i18n file written in this style

Reply:
100% agree on that. I try to refactor files when I touch them. However I defer this task a bit, but I'll do it.